### PR TITLE
adds 5-get-all-articles

### PR DESCRIPTION
**Pull request by sirbacharach**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -34,7 +34,31 @@ describe("GET /api/topics", () => {
...
+  });
+});
+
+describe("GET /api/articles", () => {
+  test("200: responds with an array of all articles in order of date descending", () => {
+    return request(app)
+    .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.allArticles).toHaveLength(37);
```
Comment (mezzDavies) [inline __tests__/app.test.js]: This is a lot of articles. Are you using the test data for your tests? 👀

File: __tests__/app.test.js [hunk 2]
```diff
@@ -34,7 +34,31 @@ describe("GET /api/topics", () => {
...
+        body.allArticles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String),
```
Comment (mezzDavies) [inline __tests__/app.test.js:59]: 👍

**Review by mezzDavies — APPROVED** [review]
This looks good but you should look at what data you are passing to the seed in the test file.

File: __tests__/app.test.js (hunk 1 above)
Comment (sirbacharach) [reply __tests__/app.test.js]: Thanks, I've altered it to the test file, I noticed this on another function that I was working on whilst I was waiting for the pull request to be checked and I've altered it on that one when I noticed too.

**Pull request merged**